### ISSUE TITLE
cdnjs.cloudflare.com delivering 1.1.0 instead of 1.2.1

### ISSUE THREAD

**Issue by drlippman** [jsxgraph/jsxgraph#341]
Certain functions aren't showing with a `functiongraph`.  

Minimal reproduction example:
````
var board = JXG.JSXGraph.initBoard('jxgboard', {boundingbox: [-5,5,5,-5], axis:true});
var f = function(x) {  return 2*(x-3)*(x+1); }
var graph = board.create('functiongraph', [f]);
````

Expected behavior: a graph of the function
Actual behavior:  a blank grid
No javascript errors are logged

The graph will appear if the function is changed to `2.1*(x-3)*(x+1)`, so it seems to be some specific characteristic of that function.

This issue does not exist in 0.99.7, but does starting in 1.0.0, and still exists in 1.2.1.

**Comment by alfredwassermann** [comment]
I can confirm this bug for 1.0.0 and 1.1.0, but not for 1.2.1 (and 1.2.0), see https://jsfiddle.net/oe01k4f6/
Can provide a jsfiddle example?
Best wishes,
Alfred

**Comment by drlippman** [comment]
See https://jsfiddle.net/dlippman/fq9n0kju/

Perhaps the cdnjs.cloudflare.com repo isn't loading the right version?  Ahh, yeah, it appears it's delivering 1.1.0 instead of 1.2.1.

**Comment by drgrice1** [comment]
I am still seeing this with some graphs with version 1.2.1 (not the incorrect version 1.1.0 from cloudflare).
Here is a minimal example:
```javascript
var board = JXG.JSXGraph.initBoard('box', { boundingbox: [-8, 1600, 8, -1600], });
board.create('curve', [
	function(t) { return t; },
	function(t) { return Math.pow(t, 2) * (t + 5) * Math.pow(t - 5, 2); },
	-8, 8
]);
```
With that example no graph appears.  If you remove the -8, 8 domain or change that to -7.95, 7.95 the graph appears, but there are messages in the console of the form "TODO _findStartPoint ..."

**Comment by alfredwassermann** [comment]
OK, you are right. Thanks for your patience.
I removed this part of `_findStartPoint`, because it is too unstable numerically.
It will be available in today's nightly build. 
You also might experiment with the upcoming new plotting algorithm, `plotVersion:4`:

    board.create('curve', [
      function(t) { return t; },
      function(t) { return Math.pow(t, 2) * (t + 5) * Math.pow(t - 5, 2); },
      -8, 8], 
      {plotVersion: 4});

Version 4 of the plot algorithm is not affected by this bug.
Best wishes, Alfred

**Comment by drgrice1** [comment]
I will look into the new plot algorithm.
Thanks.

**Comment by alfredwassermann** [comment]
Well, it still has its shortcomings. But it has a promising fresh approach.

**Comment by alfredwassermann** [comment]
The bug in the title is now fixed since 1.2.2. It was a bug in the update process.

**Comment by drgrice1** [comment]
Sounds good.  Thanks.
